website/addons/wiki/static/pagedown-ace/Markdown.Editor.js

```diff
@@ -1837,14 +1837,15 @@ var Range = ace.require('ace/range').Range;
     };
 
     commandProto.addLinkDef = function (chunk, linkDef) {
-
+        

```

Comment:
whoops, I can fix the whitespace and comments that were accidentally left in and update the PR now